WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -134,7 +145,7 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
         case StatsSectionInsightsMostPopular:
             return 2;
         case StatsSectionInsightsTodaysStats:
-            return IS_IPAD ? 2 : 5;
+            return self.isViewHorizontallyCompact ? 5 : 2;
```

Comment:
Just wanted to note (as we already discussed over in Slack!) that `isViewHorizontallyCompact` should [return *false* for iPhone 6+ devices](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LayoutandAppearance.html), so we're not maintaining exactly the same logic.

Nevertheless, i did test this, and looks awesome!